apiserver/routers/routers.go

```diff
@@ -12,8 +12,24 @@
 //    License for the specific language governing permissions and limitations
 //    under the License.
 
+// Package routers Garm API.
+//
+// The Garm API generated using go-swagger.
+//
+//	Schemes: http, https
+//	Host: localhost:9997
```

Comment:
Apparently we can [omit the host](https://swagger.io/docs/specification/2-0/api-host-and-base-path/#omit) and have that be filled in dynamically. A potential `codegen` tool or the swagger UI will use whatever host is used to access the definitions.

Reply:
Ok. Thanks for pointing out.